application/classes/Ushahidi/Repository/Contact.php

```diff
@@ -132,8 +132,10 @@ public function isInTargetedSurvey($contact_id)
     {
```

Comment:
For clarity, could we call this isInActiveTargetedSurvey, since I think that's really what we're trying to differentiate?